postgres/defaults.yaml

```diff
@@ -39,3 +39,30 @@ postgres:
     command: service postgresql initdb
     test: test -f /var/lib/pgsql/data/PG_VERSION
     env: {}
+
+  # Client executables list
+  client_bins:
+    - clusterdb
+    - createdb
+    - createlang
+    - createuser
+    - dropdb
+    - droplang
+    - dropuser
+    - pg_archivecleanup
+    - pg_basebackup
+    - pg_config
+    - pg_dump
+    - pg_dumpall
+    - pg_isready
+    - pg_receivexlog
+    - pg_restore
+    - pg_rewind
+    - pg_test_fsync
+    - pg_test_timing
+    - pg_upgrade
+    - pg_xlogdump
```

Comment:
These binaries `pg_rewind
pg_test_fsync
pg_test_timing
pg_upgrade
pg_xlogdump
` do not exist in my Debian's installation, so I don't think this list belongs into the 'default.yaml' file, it seems to be specific to the upstream repo.